.azure-pipelines/sync-aliases.yml

```diff
@@ -0,0 +1,100 @@
+resources:
+  repositories:
+  - repository: self
+    type: git
+    ref: main
+
+trigger: none
+
+variables:
+  GithubToken: $(GITHUB_TOKEN)
+  
+jobs:
+- job: UpdateJson
+  displayName: Update Json
+  steps:
+  - checkout: self
+    persistCredentials: true
+  - task: PowerShell@2
+    displayName: Update json file
+    inputs:
+      targetType: inline
+      script: |
+        $username="v-hongtzhang"
+        $password='$(AZURE-PASS)'
+        $pair="{0}:{1}" -f ($username,$password)
+        $bytes = [System.Text.Encoding]::ASCII.GetBytes($pair)
+        $token = [System.Convert]::ToBase64String($bytes)
```

Comment:
This is a bad idea. This password is too important you shouldn't trust anyone to store it.

Wiki is also a repo, maybe you can clone the repo instead?